packages/neutrino-preset-web/index.js

```diff
@@ -30,7 +30,10 @@ module.exports = (neutrino, opts = {}) => {
     polyfills: {
       async: true
     },
-    babel: {}
+    babel: {},
+    clean: {
```

Comment:
It would also be nice to disable cleaning by passing `false` to `options.clean`. You should be able to achieve this by changing this to:

```js
clean: opts.clean !== false && {
  paths: [neutrino.options.output]
}
```